src/dist/replication/lib/replica_2pc.cpp

```diff
@@ -74,6 +74,25 @@ void replica::on_client_write(dsn::message_ex *request, bool ignore_throttling)
         return;
     }
 
+    if (_is_bulk_load_ingestion) {
+        // reject write requests during ingestion
+        response_client_write(request, ERR_BUSY);
+        return;
+    }
+
+    if (request->rpc_code() == dsn::apps::RPC_RRDB_RRDB_BULK_LOAD) {
+        ddebug_replica("receive bulk load ingestion request");
+
+        // bulk load ingestion request requires that all secondaries should be alive
+        if (static_cast<int>(_primary_states.membership.secondaries.size()) + 1 <
+            _primary_states.membership.max_replica_count) {
+            response_client_write(request, ERR_NOT_ENOUGH_MEMBER);
```

Comment:
How about add a log here?

Reply:
For bulk load request, response_client_write will response to meta server, meta server will print error log, it is okay that no log here.